apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -586,54 +586,12 @@ function wpcom_is_previewing_global_styles( ?int $user_id = null ) {
  * @param  int $blog_id Blog ID.
  * @return bool Whether the site has access to Global Styles with a Personal plan.
  */
-function wpcom_site_has_global_styles_in_personal_plan( $blog_id = 0 ) {
+function wpcom_site_has_global_styles_in_personal_plan( $blog_id = 0 ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
If `treatment` wins, ideally we should remove the `wpcom_site_has_global_styles_in_personal_plan` function and add `WPCOM_Features::WPCOM_PERSONAL_PLANS` to the `WPCOM_Features::GLOBAL_STYLES` feature.

That way, `wpcom_site_has_feature( WPCOM_Features::GLOBAL_STYLES )` returns true.